app/version.properties

```diff
@@ -1,3 +1,3 @@
-BUILD_VERSION=21
+BUILD_VERSION=22
```

Comment:
It will be incremented by CI
In case the number incremented without your changes, please contact me

Reply:
Fixed